src/Uno.UI.FluentTheme/Resources/Priority02/MenuFlyout_themeresources.xaml

```diff
@@ -639,21 +639,21 @@
                             Grid.Column="0"
                             HorizontalAlignment="Left"
                             Margin="0">
-                            <FontIcon FontFamily="Segoe MDL2 Assets" Glyph="&#xE8C1;" />
+							<FontIcon FontFamily="{ThemeResource SymbolThemeFontFamily}" Glyph="&#xE8C1;" />
```

Comment:
Changes from spaces to tabs will look strange in XAML several places here. Personally, I would keep spaces in XAML as it follows upstream. Eventually I expect XAML will be copy-pasted almost verbatim.

Reply:
@robloo Visual Studio formatted this based on .editorconfig. I'll revert back to spaces and will adjust .editorconfig